zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -38,6 +38,10 @@ class EcommerceCrawlStrategy(str, Enum):
     """Follow pagination and product detail pages. Subcategory links are
     ignored."""
 
+    direct_product: str = "direct_product"
```

Comment:
I'm thinking if we should have this in the Base strategies since it could also be applicable to the upcoming articles. Maybe call it `direct_item` or something?

Reply:
Renaming to direct_item makes sense to me, because other strategies also use the same names across strategies (e.g. full vs full_product).

Reply:
I'm not sure that it'd be possible to have it in some kind of Base params or Base stategy, as spiders are different, but anyways, this can be addressed separately if needed.